js/galleryfileaction.js

```diff
@@ -210,6 +216,13 @@ $(document).ready(function () {
 	var url = window.galleryFileAction.buildGalleryUrl('config', '', {extramediatypes: 1});
 	$.getJSON(url).then(function (config) {
 		window.galleryFileAction.buildFeaturesList(config.features);
+
+		if (config.mediatypes.length === 0) {
+			config.mediatypes = ['image/png', 'image/jpeg', 'image/gif', 'image/x-xbitmap', 'image/bmp'];
```

Comment:
bitmap does not work well because of a media type mismatch between oC and PHP, so I would remove it, but that's not very important.